multimodal/setup.py

```diff
@@ -28,35 +28,33 @@
     "tqdm",
     "boto3",
     "requests",
-    "jsonschema<=4.8.0",
-    "seqeval<=1.2.2",
-    "evaluate<=0.3.0",
-    "accelerate>=0.9,<0.16",
+    "jsonschema<4.18",
+    "seqeval<1.2.3",
+    "evaluate<0.4.0",
+    "accelerate>=0.9,<0.17",
     "timm<0.7.0",
     "torch>=1.9,<1.14",
     "torchvision<0.15.0",
     "fairscale>=0.4.5,<0.4.14",
     "scikit-image>=0.19.1,<0.20.0",
-    "smart_open>=5.2.1,<5.3.0",
     "pytorch_lightning>=1.8.0,<1.10.0",
-    "text-unidecode<=1.3",
+    "text-unidecode<1.4",
     "torchmetrics>=0.8.0,<0.9.0",
-    "transformers>=4.23.0,<4.26.0",
-    "nptyping>=1.4.4,<1.5.0",
-    "omegaconf>=2.1.1,<2.2.0",
+    "transformers>=4.23.0,<4.27.0",
+    "nptyping>=1.4.4,<2.5.0",
+    "omegaconf>=2.1.1,<2.3.0",
     "sentencepiece>=0.1.95,<0.2.0",
     f"autogluon.core[raytune]=={version}",
     f"autogluon.features=={version}",
     f"autogluon.common=={version}",
-    "pytorch-metric-learning>=1.3.0,<1.4.0",
+    "pytorch-metric-learning>=1.3.0,<2.0",
```

Comment:
@zhiqiangdon I noticed that pytorch-metric-learning released version 2.0. Do we need to consider to update the version?

Reply:
I [checked](https://github.com/conda-forge/autogluon.multimodal-feedstock/pull/15/commits/8164e39502a87c0ea265145340d70e4d0e6f5553) and `<2` would be enough to get things working. It's not ideal and should be lifted to allow v2, but it's less urgent than lifting the cap from `<1.4`.

Reply:
pytorch-metric-learning seems okay to upgrade.